blocks/get-a-quote-form/get-a-quote-form.css

```diff
@@ -13,6 +13,12 @@
 }
 
 .get-a-quote-form > .content {
+  display: grid;
   background-color: var(--color-gray);
   color: transparent;
 }
+
+.get-a-quote-form > .content > .form-container  {
+  margin: 0;
+  width: 100%;
```

Comment:
There seems to width issue in overall container on checking 
https://inline-form--shredit--stericycle.aem.page/en-us/service-locations/tucson

Reply:
<img width="1728" alt="Screenshot 2024-10-08 at 15 57 27" src="https://github.com/user-attachments/assets/df38b6a6-de2d-4255-abd4-aa6eab53ccbb">
The scrollbar is coming for the form as well